fc3/myjson.py

```diff
@@ -10,14 +10,15 @@
 
 class JsonResponse(HttpResponse):
     def __init__(self, data):
-        HttpResponse.__init__(self, json_encode2(data), mimetype='text/javascript')
+        super(JsonResponse, self).__init__(json_encode2(data),

```

Comment:
This is the canonical way to invoke the super-class method of the same name, rather than directly referencing the parent class (in this case HttpResponse.) Why? Suppose you changed the inheritance to `class JsonResponse(SomeOtherClass)`. The code would still be invoking `HttpResponse.__init__()` instead of `SomeOtherClass.__init__()`.